Data/Ascii/AsciiDataIO.hpp

```diff
@@ -28,11 +28,11 @@ namespace Ascii {
 /// \f$\pi^-\f$, then one for the \f$\gamma\f$, and finally back to \f$\pi^+\f$.
 /// You may choose to start each event group with a weight value, but you don't
 /// need to.
-std::vector<ComPWA::Event> readData(const std::string &InputFilePath,
-                                    long long NumberEventsToRead = -1);
+EventCollection readData(const std::string &InputFilePath,
```

Comment:
Just a small remark (just in case you are not aware of it): I (try to) use arguments by copy in case the code is non performance critical, and in many cases the compiler can optimize the copy with a move anyways.